lib/extensions/conditional/active_record/base.rb

```diff
@@ -32,6 +32,18 @@ def specific_conditions
     def conditions_satisfied_by?(course_user)
       conditions.all? { |condition| condition.satisfied_by?(course_user) }
     end
+
+    # Unlock the conditional for the given course user.
+    #
+    # @param [CourseUser] _course_user The course user in which the conditional is to unlock for
+    def unlock_for(_course_user)
```

Comment:
lol, use_case.

I think conditions and conditionals should be fine because @custode's interim report details the terminology very clearly... the dependency graph is instantiated into a satisfiability graph to be evaluated in the context of a user.